### DISCUSSION TITLE
DNN Not Supported OR Not Subscribed in the Slice

### DISCUSSION THREAD

**Discussion by rolandmartinson7890** [srsran/srsRAN_Project#268] [General Help] [answered]
Has anyone every try to work with Quectel RM500Q-GL model. I always find it unstable.

Here is the error I am experiencing. Does anyone know how to solve this? My APN on Open5gs matches the one I configured on the Quectel Card. 

10/23 18:09:34.029: [amf] INFO: gNB-N2[127.0.0.1] max_num_of_ostreams : 30 (../src/amf/amf-sm.c:780)
10/23 18:10:28.848: [amf] INFO: InitialUEMessage (../src/amf/ngap-handler.c:401)
10/23 18:10:28.848: [amf] INFO: [Added] Number of gNB-UEs is now 1 (../src/amf/context.c:2523)
10/23 18:10:28.848: [amf] INFO:     RAN_UE_NGAP_ID[0] AMF_UE_NGAP_ID[1] TAC[1] CellID[0x19b0] (../src/amf/ngap-handler.c:562)
10/23 18:10:28.849: [amf] INFO: [suci-0-001-01-0-0-0-0123456792] Unknown UE by SUCI (../src/amf/context.c:1789)
10/23 18:10:28.849: [amf] INFO: [Added] Number of AMF-UEs is now 1 (../src/amf/context.c:1570)
10/23 18:10:28.849: [gmm] INFO: Registration request (../src/amf/gmm-sm.c:1061)
10/23 18:10:28.849: [gmm] INFO: [suci-0-001-01-0-0-0-0123456792]    SUCI (../src/amf/gmm-handler.c:157)
10/23 18:10:29.155: [gmm] INFO: [imsi-001010123456792] Registration complete (../src/amf/gmm-sm.c:1993)
10/23 18:10:29.155: [amf] INFO: [imsi-001010123456792] Configuration update command (../src/amf/nas-path.c:612)
10/23 18:10:29.155: [gmm] INFO:     UTC [2023-10-23T16:10:29] Timezone[0]/DST[0] (../src/amf/gmm-build.c:558)
10/23 18:10:29.155: [gmm] INFO:     LOCAL [2023-10-23T18:10:29] Timezone[7200]/DST[1] (../src/amf/gmm-build.c:563)
10/23 18:10:29.155: [amf] INFO: [Added] Number of AMF-Sessions is now 1 (../src/amf/context.c:2544)
10/23 18:10:29.155: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:10:45.088: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:11:01.088: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:11:17.088: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:11:33.088: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:12:49.128: [gmm] WARNING: [imsi-0010101DNN Not Supported OR Not Subscribed in the Slice23456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:13:05.128: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:13:21.128: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:13:37.127: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)
10/23 18:13:53.127: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)

**Comment by rolandmartinson7890** [comment]
I am just observing that it is a similar issue here as per amf logs of the Quectel: https://github.com/srsran/srsRAN_Project/discussions/189. I see that the quectel is on the list of successfully connected UEs with srsRAN Project (https://docs.srsran.com/projects/project/en/latest/knowledge_base/source/cots_ues/source/index.html). Was this problem ever faced?

**Comment by herlesupreeth** [comment]
The answer is in your logs :)

> 10/23 18:10:29.155: [gmm] WARNING: [imsi-001010123456792] DNN Not Supported OR Not Subscribed in the Slice (../src/amf/gmm-handler.c:1234)

Check the APN configured on your modem and open5gs Web UI for this IMSI.

**Reply by rolandmartinson7890** [reply]
Thanks for the reply. The APNs are matching. I mentioned that in the question. So I am wondering how people ran their AT commands to overcome this.

**Reply by herlesupreeth** [reply]
My bad then. Can you please post a pcap of NGAP interface?

Also, I use mmcli to initiate the PDN connection. For example:

```
# Lists the modems connected to the machine
sudo mmcli -L
# Initiates the PDN connection for srsapn
sudo mmcli -m <modem_number> --simple-connect="apn=srsapn"
```

**Reply by pgawlowicz** [reply]
@rolandmartinson7890 any update on this issue?

**Reply by hakegard** [reply, marked as answer]
We get the same warning using a Simcom modem. The reason is that by default, the PDU services Internet, IMS and SOS are defined in the modem (seen by the AT+CDGCONT command). The modem connects without any problem to the Internet service, but still apparently polls for the IMS and/or SOS services so that the warning occurs in the log. We just ignore the warning, but it can be made  disappear by removing the IMS and SOS services from the modem configuration.
